TeachingRecordSystem/tests/TeachingRecordSystem.TestCommon/CrmTestData.CreatePerson.cs

```diff
@@ -179,6 +208,16 @@ public async Task<CreatePersonResult> Execute(CrmTestData testData)
                 contact.dfeta_NINumber = testData.GenerateNationalInsuranceNumber();
             }
 
+            if (_qtsDate is not null)
+            {
+                contact.dfeta_QTSDate = _qtsDate.Value.FromDateOnlyWithDqtBstFix(isLocalTime: true);
+            }
+
+            if (_eytsDate is not null)
+            {
+                contact.dfeta_EYTSDate = _eytsDate.Value.FromDateOnlyWithDqtBstFix(isLocalTime: true);
+            }
+
```

Comment:
This won't work with the real CRM; these fields are computed from `qtsregistration` entities for the `contact` via a plugin. We will have to port/emulate the plugin so this method will still work for the real CRM.